internal/ingress/annotations/redirect/redirect_test.go

```diff
@@ -32,6 +32,7 @@ import (
 
 const (
 	defRedirectURL = "http://some-site.com"
+	defTrue        = "true"
```

Comment:
I don't think this is a good name for this variable. Please be more concrete about what this is the default for in the variable name.

Or just keep it as is and use a hard-coded `"true"`.

Reply:
It is not. I mean - who does that.
 I did it to make the linter happy. 
To keep the linter happy I could rename it to sth like `defAnnotationTrueValue` ?

Reply:
`defAnnotationValueTrue`
ok?

Reply:
The name should be about where its value is being used, not its value. Even if this means having multiple variables with different values all set to true.

Reply:
ok. I think the name reflects that